x-pack/plugins/upgrade_assistant/server/routes/ml_snapshots.ts

```diff
@@ -256,7 +260,7 @@ export function registerMlSnapshotRoutes({ router, lib: { handleEsError } }: Rou
                     'xpack.upgradeAssistant.ml_snapshots.modelSnapshotUpgradeFailed',
                     {
                       defaultMessage:
-                        "The upgrade that was started for this model snapshot doesn't exist anymore.",
+                        'The upgrade process for this model snapshot failed. Check the Elasticsearch logs for more details.',
```

Comment:
I wonder if it wouldn't be good to add metadata with the error message and the `log()` call you added below.

Something like

```
`The upgrade process for this model [${modelId}] failed...`
```

And maybe the snapshot id, job id if it make sense to include them?

Reply:
Thanks for the review! I don't think it adds of much since we will know which job/snapshot failed after kicking the upgrade since the UI and the api call will tell us all we need

Reply:
If an admin is looking at the logs of a large deployment I think it's unlikely to have access to the UI or the original API call at the time the problem occurs. Would this info be useful in that case?

Reply:
That's a good point @cjcenizal! Ill patch it up in a bit!

Reply:
Yes that's what I thought. If there is an SDH case coming up we'll only have the logs to look at. The more there is there the better 😊